src/index.js

```diff
@@ -1,14 +1,27 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import reportWebVitals from './reportWebVitals';
-import theme from "./utils/theme"
+import theme from "./utils/theme";
 import {MuiThemeProvider, CssBaseline } from "@material-ui/core";
-import Card from './components/card/card';
+import { Route, Switch } from 'react-router-dom';
+import { BrowserRouter as Router } from 'react-router-dom';
+import routes from './routes';
 
 ReactDOM.render(
   <MuiThemeProvider theme={theme}>
     <CssBaseline />
-      <Card />
+    <Router basename = {"ecommerce"} >
```

Comment:
porque usar um basename?

Reply:
Porque ele é o URL base para todos os locais ?
Você não usaria ? 

Obs: basename: string
The base URL for all locations. A properly formatted basename should have a leading slash, but no trailing slash.